suite/tests/api/drdecode_aarch64.c

```diff
@@ -82,6 +82,8 @@ test_mov_instr_addr(void)
     instrlist_encode(GD, ilist, generated_code, true);
```

Comment:
Regarding the PR's description and title: please take a look at the section on Commit Messages at https://dynamorio.org/page_code_reviews.html. Particularly, please shorten the PR's title, and add more details about the bug and the issue number to the description.

Reply:
Thanks for the changes. Note that conventions also suggest a `#` in the issue number in the title, i.e. "i#5033: Fix intermittent..."